sigle/src/modules/analytics/stats/StatsError.tsx

```diff
@@ -0,0 +1,28 @@
+import { CrossCircledIcon } from '@radix-ui/react-icons';
```

Comment:
Just a note but we should see with Quentin to make the alerts part of the design-system so we can reuse them easily on other views if needed. Not to do now, but smth to think about for the future.

Reply:
Do you mind creating an issue so we don't forget about this one please? :)